### ISSUE TITLE
Whenever i train the YOLOv8 with my custom dataset repeatedly, loss value increases. What is the reason? 

### ISSUE THREAD

**Issue by DGUnggi** [pjreddie/darknet#2639]
!pip install ultralytics
!pip install roboflow

from roboflow import Roboflow
rf = Roboflow(api_key="~")
project = rf.workspace("~").project("~")
version = project.version(12)
dataset = version.download("yolov8")

!pip install PyYAML

import yaml

data = {'train' : '/workspace/ExDark-12/train/images/',
        'val' : '/workspace/ExDark-12/valid/images/',
        'test' : '/workspace/ExDark-12/test/images/',
        'names' : ['Bicycle','Boat','Bottle','Bus','Cat','Cup','Motorbike','People','Table','car','chair','dog'],
        'nc' : 12}

with open('/workspace/ExDark-12/data.yaml', 'w') as f:
  yaml.dump(data, f)

with open('/workspace/ExDark-12/data.yaml', 'r') as f:
  exdark_yaml = yaml.safe_load(f)
  display(exdark_yaml)

`model = YOLO('/workspace/runs/detect/BestYolo/weights/best.pt')`   - (1)
`model.train(data = '/workspace/ExDark-12/data.yaml', epochs = 13, batch = 32, imgsz = 640, name = "BestYolo", exist_ok = True )`  -(2)

I want to train the model(1) with 5 epochs, 10 epochs, and 50 epochs. But whenever i retrain the model, the loss value increases.
Below is the example.

![image](https://github.com/pjreddie/darknet/assets/129149629/92dc7e3e-58d1-41ad-bde2-2d569e5062ba)
